src/index.tsx

```diff
@@ -398,12 +398,14 @@ export function useCamera() {
  * This hook can toggle microphone on/off and provides current microphone state.
  */
 export function useMicrophone() {
-  const [isMicrophoneOn, setIsMicrophoneOn] = useState<boolean>(false);
+  const [isMicrophoneOn, setIsMicrophoneOn] = useState<boolean>(
+    MembraneWebRTCModule.isMicrophoneOn
+  );
 
   useEffect(() => {
     const eventListener = eventEmitter.addListener<IsMicrophoneOnEvent>(
       'IsMicrophoneOn',
```

Comment:
I was wondering is it a good idea to create a script which generates swift, kotlin and react  files based on json config, that will get rid off discrepancy between events naming

config could look like:
{
  "MAMA" : "MAMA",
  "TATA" : "TATA",
  "BRAT" : "BRAT"
}
 and script ( in python for example):
 
 import json

with open('variables.json', 'r') as f:
    variables = json.load(f)

with open('variables.ts', 'w') as f:
    for key, value in variables.items():
        f.write(f'const {key} = "{value}";\n')

with open('variables.swift', 'w') as f:
    for key, value in variables.items():
        f.write(f'let {key} = "{value}"\n')

with open('variables.kt', 'w') as f:
    f.write('object Constants {\n')
    for key, value in variables.items():
        f.write(f'    val {key} = "{value}"\n')
    f.write('}\n')